src/features/common/TreeCounter/TreeCounter.tsx

```diff
@@ -64,81 +88,63 @@ export default function TpoProfile(props: any) {
+      )}
 
-        {props.target ? (
-          <div className={treeCounterStyles.treeCounterDataField}>
-            <h1>
-              {localizedAbbreviatedNumber(
-                i18n.language,
-                Number(props.target),
-                1
-              )}
-            </h1>
-            <div className={treeCounterStyles.target}>
-              <h2
-                className={
-                  props.authenticatedType === 'public'
-                    ? treeCounterStyles.targetText
-                    : treeCounterStyles.targetTextForPrivate
-                }
-              >
-                {t('me:target')}
-              </h2>
-              {props.authenticatedType === 'private' && (
-                <button
-                  id={'treeCounterEdit'}
-                  className={treeCounterStyles.editTragetContainer}
-                  onClick={() => props.handleAddTargetModalOpen()}
-                >
-                  <EditIcon color="white"></EditIcon>
-                </button>
-              )}
-            </div>
-          </div>
-        ) : null}
+      <div
+        className={
+          isHomeTreeCounter
+            ? treeCounterStyles.backgroundCircle
+            : treeCounterStyles.backgroundCircleForTenant
+        }
+      />
 
-        {props.authenticatedType === 'private' && props.target === 0 && (
-          <button
-            id={'addTarget'}
-            onClick={() => props.handleAddTargetModalOpen()}
-            className={treeCounterStyles.addTargetButton}
-          >
-            <p
-              style={{
-                color:
-                  theme === 'theme-light'
-                    ? themeProperties.light.light
-                    : themeProperties.dark.dark,
-              }}
-            >
-              {t('me:addTarget')}{' '}
-            </p>
-          </button>
-        )}
-      </div>
+      {isHomeTreeCounter ? (
+        <HomeTreeCounter planted={props?.planted} target={props.target} />
+      ) : (
+        <div className={treeCounterStyles.treeCounterData}>
+          <div>
+            <PlantedTressBlackSvg color={'#4F4F4F'} />
+          </div>
+          <div className={treeCounterStyles.dataContainer}>
+            {props?.planted && (
+              <div>
+                {localizedAbbreviatedNumber(
+                  i18n.language,
+                  Number(props.planted - props.restoredAreaUnit),
+                  1
+                )}
+              </div>
+            )}
+            {props.target !== undefined && props.target !== 0 && (
+              <div>{'of'}</div>
+            )}
+            {props?.target !== 0 && (
+              <div>
+                {localizedAbbreviatedNumber(
+                  i18n.language,
+                  Number(props.target),
+                  1
+                )}
+              </div>
+            )}
+          </div>
+          <div style={{ fontSize: '24px' }}>{t('me:treesPlanted')}</div>
```

Comment:
Some singular / plural distinction here would be nice using a counter for the translation resource distinction. Currently this resource is still talking about a single tree in some languages and there seems to be an unused resource named `treesPlanted_plural`?

Reply:
I changed this to plural for the German language as well now.